src/coreclr/jit/hwintrinsicxarch.cpp

```diff
@@ -267,17 +267,17 @@ static CORINFO_InstructionSet lookupInstructionSet(const char* className)
         {
             if (strncmp(className + 6, "128", 3) == 0)
             {
-                assert((className[9] == '\0') || (strcmp(className + 9, "`1") == 0));
+                //assert((className[9] == '\0') || (strcmp(className + 9, "`1") == 0));
```

Comment:
Unintended change?

Reply:
See commit title https://github.com/dotnet/runtime/pull/103948/commits/f71b07387ae4fc43b01192253945b79e46f58275

Outerloop is on the floor again and I'm getting really tired being the janitor.

Reply:
GitHub UI only showed me the first two commits in the list and thus I didn't see the title. Must have been some glitch.

Reply:
A fix for this up here: https://github.com/dotnet/runtime/pull/103995/files

However, the fact that the assert triggered looks to be a bug in NAOT, as there are types being marked as `[Intrinsic]` which should not be marked `[Intrinsic]` and which are not marked as such in the actual managed implementation.